src/utils.js

```diff
@@ -151,13 +118,19 @@ const getTripRange = (events) => {
   };
 };
 
-const getDestinations = (events) =>
-  Array.from(new Set(events.map(({ destination }) => destination)));
+const getDestinations = (events) => {
+  const start = events[0].destination;
+  const end = events[events.length - 1].destination;
+
+  return [ start, ...Array.from(new Set(events.slice(1, events.length - 1).map(({ destination }) => destination))), end];
+};
+
 
 const filterEvents = {
```

Comment:
у нас это структура данных мап (словать)
Должно быть соотвествующие название
```suggestion
const filterTypeToSortCb = {
```

Reply:
Исправила